### PR TITLE
Create vim doc page (finally)

### PR DESCRIPTION
**Pull request by pappasam**
Created a doc page based on the existing markdown README. This addresses the HELP request at papercolor-theme/issues/115.

I've also added a gitignore to the repository. The documentation page generates "tags" which should be ignored from version control.

### CODE REVIEW

**Comment by pappasam** [comment]
@NLKNguyen any thoughts on this PR?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#################################################################
+# Global/Vim.gitignore
+#################################################################
+# Swap
+[._]*.s[a-v][a-z]
+[._]*.sw[a-p]
+[._]s[a-rt-v][a-z]
+[._]ss[a-gi-z]
+[._]sw[a-p]
```
Comment (pickfire) [inline .gitignore]: Why is that needed in this repository?

**Comment by NLKNguyen** [comment]
Awesome. Sorry for late response. Thanks a lot for helping me out!!!

File: .gitignore (hunk 1 above)
Comment (NLKNguyen) [reply .gitignore]: Yeah, I'm curious too.

Comment (pappasam) [reply .gitignore]: Hmm, good question! tbh, just added the gitignore from GitHub's gitignore templates: https://github.com/github/gitignore/blob/master/Global/Vim.gitignore

If you'd prefer shortening the gitignore to include only the patterns we fully understand, I'm on board with that.
